o2.sh

```diff
@@ -172,12 +173,13 @@ module load BASE/1.0 FairRoot/$FAIRROOT_VERSION-$FAIRROOT_REVISION ${DDS_VERSION
 # Our environment
 setenv O2_ROOT \$::env(BASEDIR)/$PKGNAME/\$version
 setenv VMCWORKDIR \$::env(O2_ROOT)/share
-prepend-path PATH \$::env(O2_ROOT)/bin
-prepend-path LD_LIBRARY_PATH \$::env(O2_ROOT)/lib
+
+set O2_ROOT \$::env(O2_ROOT)
+prepend-path PATH \$O2_ROOT/bin
+prepend-path LD_LIBRARY_PATH \$O2_ROOT/lib
 $([[ ${ARCHITECTURE:0:3} == osx && ! $BOOST_VERSION ]] && echo "prepend-path ROOT_INCLUDE_PATH $BOOST_ROOT/include")
```

Comment:
@ktf this line doesn't pass the linter prepen-path check. Do you maybe have a suggestion how to fix it?